style.css

```diff
@@ -0,0 +1,410 @@
+  width: 70%;
+  background-color: var(--orange);
+  font-size: 2rem;
+}
+
+.profile {
+  width: 40%;
+}
+
+.idea {
+  width: 60%;
+  color: #4c4c4c;
+}
+
+.img1 {
+  height: 9rem;
+  width: 9rem;
+}
+
+.img2 {
+  width: 95%;
+  height: 90%;
+}
+
+.z-3 {
+  z-index: 3;
+}
+
+.speaker-underline {
+  background-color: var(--orange);
+  width: 3rem;
+  padding: 0.1rem 3rem;
+}
+
+.space {
+  background-color: var(--bg);
+  padding: 0.1rem 2rem;
+  width: 1rem;
+}
+
+/*               adding thickness to chevron icon            */
+.p path {
+  stroke: var(--orange);
+  stroke-width: 2px;
+}
+
+/* ******************** popup section *************/
+
+.popup {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  background-color: rgba(211, 211, 211, 0.8);
+  z-index: 5;
+}
+
+.popup-content {
+  height: 100%;
+  width: 65%;
+  margin-right: auto;
+  background-color: rgb(39, 42, 49);
+}
+
+.humberger2 {
+  margin: 1rem;
+}
+
+.span-1 {
+  transform: rotate(-38deg) translate(-1rem, 0.5rem);
+  width: 1.6rem;
+}
+
+.span-2 {
+  transform: rotate(38deg) translate(-1rem, -0.5rem);
+}
+
+.hide,
+.no-display {
+  opacity: 0;
+}
+
+.links {
+  width: 90%;
+  margin: auto;
+  padding-top: 3rem;
+}
+
+.links ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.links li {
+  margin: 2rem 0;
+}
+
+.links a {
+  text-decoration: none;
+  color: white;
+  font-size: 4rem;
+  padding: 1rem;
+}
+
+.links a:hover {
+  background-color: var(--bg);
+  color: var(--orange);
+}
+
+/* *******************  about page **********/
+.layer {
+  background-color: rgba(109, 27, 18, 0.6);
+}
+
+.partners {
+  background-color: var(--gray);
+  padding-top: 3rem;
+  padding-bottom: 3rem;
+}
+
+.w-30 {
+  width: 20rem;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
+}
+
+.w-30 img {
+  width: 100%;
+}
+
+.footer-logo {
+  width: 70%;
+}
+
+.hide-mob {
+  display: none;
+}
+
+.grid-speakers {
+  display: none;
+}
+
+.people {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  justify-content: center;
+}
+
+/* Media query  */
+
+@media screen and (min-width: 768px) {
+  .hide-mob {
+    display: flex;
+  }
+
+  .hide-desk {
+    display: none;
+  }
+
+  .main-head {
+    margin: 0;
+  }
+
+  nav,
+  .navs {
+    width: 100%;
+    padding: 0.1rem 10%;
+    background-color: rgb(48, 48, 48);
+  }
+
+  li:active {
+    border-bottom: 2px solid var(--orange);
+  }
+
+  nav li:hover,
+  .navs li:hover {
+    transform: scale(1.2);
+  }
+
+  a {
+    text-decoration: none;
+    color: white;
+  }
+
+  .navs {
+    height: 8rem;
+  }
+
+  .navs img:hover {
+    transform: scale(1.1);
+  }
+
+  .logo-head {
+    width: 6rem;
+  }
+
+  .linklist a {
+    color: #272a31;
+    font-size: 2rem;
+  }
+
+  .text {
+    width: 80%;
+    padding: 8rem 0.1rem;
+  }
+
+  .text h3 {
+    font-size: 5rem;
+  }
+
+  .text-bg {
+    font-size: 7rem;
+  }
+
+  .details {
+    width: 60%;
+  }
+
+  .about {
+    padding-top: 4rem;
+  }
```

Comment:
Great job @Ali-0111 🤝

- [ ] As stated in the project requirements, kindly follow the design guidelines as much as possible. You may change your main programs section to replicate the one given in the design guidelines.
- Your design
![Your-Main-Program](https://user-images.githubusercontent.com/63917725/219649424-f338de3c-c07a-4125-a44d-620f24adc992.PNG)
- Given Design
![Given-Main-Program](https://user-images.githubusercontent.com/63917725/219649499-19420e15-06b6-45bd-b55b-50d372aeb69a.PNG)